docs/articles_en/openvino_workflow/running_inference_with_openvino/stateful_models_intro.rst

```diff
@@ -0,0 +1,121 @@
+    we can process it with RNN like models that contain a cycle inside. But in some cases, like online speech recognition of time series
+    forecasting, length of data sequence is unknown. Then data can be divided in small portions and processed step-by-step. But dependency
+    between data portions should be addressed. For that, models save some data between inferences - state. When one dependent sequence is over,
+    state should be reset to initial value and new sequence can be started.
+
+OpenVINO has a special API to simplify work with Stateful models. State is automatically saved between inferences,
+and there is a way to reset state when needed. You can also read state or set it to some new value between inferences.
+
+Stateful Model Example
+######################
+
+
+.. image:: _static/images/stateful_model_example.png
+   :align: center
+
+The left side of the picture shows the usual inputs and outputs to the model: Parameter/Result operations. They have no connection with each other and in order to copy data from output to input users need to put extra effort writing and maintaining additional code.
+In addition, this may impose additional overhead due to data representation conversion.
+
+Having operations such as ReadValue and Assign allows users to replace the looped Parameter/Result pairs of operations and shift the work of copying data to OpenVINO. After the replacement, the OpenVINO model no longer contains inputs and outputs with such names, all internal work on data copying is hidden from the user, but data from the intermediate inference can always be retrieved using State API methods.
+
+
+.. image:: _static/images/stateful_model_init_subgraph.png
+   :align: center
+
+In some cases, users need to set an initial value for State, or it may be necessary to reset the value of State at a certain inference to the initial value. For such situations, an initializing subgraph for the ReadValue operation and a special "reset" method are provided.
+
+You can find more details on these operations in [ReadValue specification](../ops/infrastructure/ReadValue_6.md) and
+[Assign specification](../ops/infrastructure/Assign_6.md).
+
+How to get OpenVINO Model with States
+#########################################
+
+* [Apply LowLatency2 transformation.](./ways_to_get_stateful_model.md#)
+   If a model contains a loop that runs over some sequence of input data,
+   the LowLatency2 transformation can be applied to get model with states.
+   Note: there are some [specific limitations]() to use the transformation.
+
+* [Apply MakeStateful transformation.](./ways_to_get_stateful_model.md)
+   If after conversion from original model to OpenVINO representation, the resulting model contains Parameter and Result operations,
+   which pairwise have the same shape and element type, the MakeStateful transformation can be applied to get model with states.
```

Comment:
Promote Optimum-Intel as the main source of stateful models. Refer to Gen AI article in this documentation. Position LowLatency2 and MakeStateful as more low-level/advanced technique for developers who really know what they want.

Reply:
done